### ISSUE TITLE
Dashboard

### ISSUE THREAD

**Issue by warrensearle** [jac-uk/admin#1669]
Creating the first iteration of a dashboard based on mock-up designed by Jeremy and additions in the doc below.

Google doc: https://docs.google.com/document/d/1DurSYDKuBL3J27Z2BgFx7trSm5A5-v-VN-tAbNjIuBM/edit

Prototype Figma file: https://www.figma.com/file/zJI7oT9EqEWQpgRWJ53lDp/JAC-Dashboard?node-id=1570%3A20151

- [x] Overview
- [x] Distribution of scores
- [x] Candidate Breadkdown at each stage
  - [x]   chart
  - [x]   table
- [x] data connection
- [x] data oprimization for the charts
  - [x]   gender
  - [x]   ethnicity
  - [x]   Current Legal Role
  - [x]   Disability
  - [x]   Social Monility

**Comment by andrew-isaac** [comment]
@warrensearle @andrew-isaac  - What is the update with this work? There was a change requested a month ago. Has this been implemented?

**Comment by warrensearle** [comment]
Hey @andrew-isaac we discussed this a couple of weeks ago in our weekly standup and apologies I meant to update this ticket but haven't until now.
The majority of changes requested here have been implemented.
There is another ticket with Dashboard feedback/changes (https://github.com/jac-uk/admin/issues/1786) which is in progress. We decided not to release the dashboard as is but instead to close this ticket and implement all remaining feedback/changes under the other ticket before releasing the dashboard.
